content/applications/sales/rental.rst

```diff
@@ -2,46 +2,123 @@
+Transfers` and :guilabel:`Digital Documents`.
+
+- :guilabel:`Default Delay Costs` are additional costs for late returns.
+- :guilabel:`Default Padding Time` represents the minimum amount of time between two rentals.
+- :guilabel:`Rental Transfers` means stock deliveries and receipts can be used for rental orders.
+- :guilabel:`Digital Documents` allows users to upload documents for customers to sign prior to
+   confirming their rental.
+
+In the :guilabel:`Rent Online` section, there are options to configure a :guilabel:`Minimal Rental
+Duration` and designate :guilabel:`Unavailability days`, or days during which pickup and return are
+not possible.
+
+Rental products
+===============
+
+To view all products that can rented in the database, navigate to :menuselection:`Rentals app -->
+Products`. By default, the :guilabel:`Can be Rented` search filter appears in the search bar.
+
+Each product kanban card displays that product's name, rental price, and product image (if
+applicable).
+
+Rental pricing
 ==============
 
-Configuration
--------------
+To adjust the rental pricing on a product, go to the :guilabel:`Products` page in the *Rental* app,
+then select the desired product or click :guilabel:`New` to create a new product from scratch.
 
-Go to :menuselection:`Rental --> Products`, select or create a product, and click on the product's
-*Rental* tab. Under *Rental Pricing*, click on *Add a price*. Then choose a *Unit* of time (hours,
-days, weeks, or months), a *Duration*, and a *Price*. You can add as many price lines as necessary,
-usually to give out discounts for longer rental durations.
+On the product form, ensure the :guilabel:`Can be Rented` checkbox is ticked. Then, open the
+:guilabel:`Rental prices` tab.
 
-.. image:: rental/rental-pricing-example.png
+.. image:: rental/rental-prices-tab.png
    :align: center
-   :alt: Example of rental pricing configuration in Odoo Rental
+   :alt: How the Settings page appears in the Odoo Rental application.
+
+.. note::
+   If creating a rental product outside of the *Rental* app, just ensure the :guilabel:`Can be
+   Rented` checkbox is ticked on the product form. By default, this checkbox is ticked whenever a
+   product is created directly in the *Rental* application.
+
+Pricing
+-------
+
+Under the :guilabel:`Pricing` section of the :guilabel:`Rental prices` tab, designate custom rental
+prices and rental periods for the product.
 
-.. tip::
-   Under *Reservations*, you can add fines for any *Extra Hour* or *Extra Day*. You can also set a
-   *Security Time*, expressed in hours, to make the product temporarily unavailable between two
-   rental orders.
+To add pricing for a rental, click :guilabel:`Add a price`. Then, choose a pricing period (the unit
+of duration of the rental) in the :guilabel:`Period` column, or create a new pricing period by
+typing in the name and clicking :guilabel:`Create`.
```

Comment:
This is a good opportunity to use the definition tag! I think we use it like this:
*Term in italics* (:dfn:`definition`)

```suggestion
To add pricing for a rental, click :guilabel:`Add a price`. Then, choose a *pricing period*
(:dfn:`the unit of duration of the rental`) in the :guilabel:`Period` column, or create a new
pricing period by typing in the name and clicking :guilabel:`Create`.
```